### PR TITLE
[Manual Search] Erase cache results when toggling scene numbering

### PR DESCRIPTION
**Pull request by fernandog**
@duramato @medariox @labrys @p0psicles

### CODE REVIEW

**Comment by duramato** [comment]
```
2016-03-03 21:50:43 Thread-15 :: [5309f79] Failed doing webui callback: Traceback (most recent call last):
  File "/home/home/SickRage/sickbeard/webserve.py", line 290, in async_call
    result = function(**kwargs)
  File "/home/home/SickRage/sickbeard/webserve.py", line 1754, in editShow
    erase_cache(showObj.indexerid)
NameError: global name 'erase_cache' is not defined
 ```

**Comment by fernandog** [comment]
@labrys help here. Whats wrong?

**Comment by fernandog** [comment]
@duramato fixed. I know check if table exists

File: sickbeard/webserve.py [hunk 1]
```diff
@@ -1750,6 +1750,25 @@ def editShow(self, show=None, location=None, anyQualities=[], bestQualities=[],
             except CantUpdateShowException as e:
                 errors.append("Unable to force an update on scene numbering of the show.")
 
+            # Must erase cached results when toggling scene numbering
+            self.erase_cache(showObj)
+
+            try:
+                # Must rename shows when toggling scene numbering
+                ui.notifications.message('Renaming all epidoses of {} after toggling scene numbering'.format(showObj.name))
```
Comment (p0psicles) [inline sickbeard/webserve.py]: Epidoses?

Comment (fernandog) [inline sickbeard/webserve.py]: lol!

**Pull request merged**